src/js/components/Layer/LayerContainer.js

```diff
@@ -137,7 +154,11 @@ const LayerContainer = forwardRef(
     }
 
     if (onEsc) {
-      content = <Keyboard onEsc={onEsc}>{content}</Keyboard>;
+      content = (
+        <Keyboard target="document" onEsc={onEsc}>
```

Comment:
This change also breaks the behavior of the Layer -> Center story when the user has opened the Select drop down and presses Esc.